bootstrap.sh

```diff
@@ -1,20 +1,21 @@
 #!/bin/bash
-kubectl apply -f .infrastructure/mysql/ns.yml
-kubectl apply -f .infrastructure/mysql/configMap.yml
-kubectl apply -f .infrastructure/mysql/secret.yml
-kubectl apply -f .infrastructure/mysql/service.yml
-kubectl apply -f .infrastructure/mysql/statefulSet.yml
+# kubectl apply -f .infrastructure/mysql/ns.yml
+# kubectl apply -f .infrastructure/mysql/configMap.yml
+# kubectl apply -f .infrastructure/mysql/secret.yml
+# kubectl apply -f .infrastructure/mysql/service.yml
+# kubectl apply -f .infrastructure/mysql/statefulSet.yml
```

Comment:
These lines are commented out, which means the MySQL resources will not be applied. Ensure this is intentional and that MySQL is either not needed or is managed separately.